samples/frameworks/react/ts/src/hooks/useEventHubs.ts

```diff
@@ -0,0 +1,97 @@
+/*
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT license.
+
+  This sample demonstrates how to create a React hook integrating
+  with Azure Event Hubs. 
+
+  For more information on Azure Event Hubs please see 
+  https://www.npmjs.com/package/@azure/event-hubs
+*/
+
+import { useEffect, useRef } from "react";
+import {
+  EventData,
+  EventHubConsumerClient,
+  EventHubProducerClient,
+  SubscriptionEventHandlers
+} from "@azure/event-hubs";
+import { credential, getEnvironmentVariable } from "../utils";
+
+const useEventHubs: (
```

Comment:
Same question as https://github.com/Azure/azure-sdk-for-js/pull/12924/files#r544678261